tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -1352,6 +1415,12 @@ public:
    }
 
    std::string GetActionName() { return "Snapshot"; }
+
+   std::function<void(unsigned int)> GetDataBlockCallback() final
+   {
+      auto callback = [this](unsigned int) mutable { fBranchAddressesNeedReset = true; };
+      return {callback};
```

Comment:
Why not just
```suggestion
      return [this](unsigned int) mutable { fBranchAddressesNeedReset = true; };
```
?

Reply:
leftover from a previous iteration where the return type was `vector<function<..>>`, thanks!